specification/programmableconnectivity/ProgrammableConnectivity.Management/OperatorApiConnection.tsp

```diff
@@ -125,43 +158,379 @@ union AccountType {
 }
 
 /**
- * Details about the Application that would use the Operator's Network APIs.
+ * Model defining how received data is processed.
  */
-model ApplicationProperties {
+@TypeSpec.Versioning.added(Versions.v2024_11_01_preview)
+model DataProcessing {
   /**
-   * Name of the application. Example: Contoso App.
+   *  The operation that is performed on the data received.
    */
-  name?: string;
+  processingOperation: ProcessingOperation;
 
   /**
-   * Description of the application.
+   * The contexts in which the data is processed.
    */
-  applicationDescription?: string;
+  contexts: Context[];
 
   /**
-   * The category that describes the application.
+   * The duration or temporal limitation of the data processing.
    */
-  applicationType?: string;
+  duration: Duration;
 
   /**
-   * Legal name of the organization owning the application.
+   * The frequency of the data processing.
+   */
+  frequency: Frequency;
+
+  /**
+   * The countries/regions the data transits through.
+   */
+  @OpenAPI.extension("x-ms-identifiers", ["countryCode"])
+  transitRegions: DataRegions[];
+
+  /**
+   * The countries/regions the data is stored in.
+   */
+  @OpenAPI.extension("x-ms-identifiers", ["countryCode"])
+  storageRegions: DataRegions[];
+}
+
+/**
+ * Model defining the information required about the countries/regions the data transits through or is stored in.
+ */
+@TypeSpec.Versioning.added(Versions.v2024_11_01_preview)
+model DataRegions {
+  /**
+   * The country code of the country/region the data transits through or is stored in.
    */
-  legalName?: string;
+  @pattern("[A-Z]{2}", "Provide 2 letter country code")
+  countryCode: string;
 
   /**
-   * A description of the organization owning the application.
+   * The commercial organization who is managing the data.
    */
-  organizationDescription?: string;
+  commercialOrganization?: string;
 
   /**
-   * Unique Tax Number for the user's organization in the country/region the APC Gateway is being purchased.
+   * Flag to indicate if commercial activity applies to this data management.
    */
-  taxNumber?: string;
+  commercialActivity: boolean;
```

Comment:
>commercialActivity: boolean; [](http://example.com/codeflow?start=2&length=28)

Recommend well-designed enum instead of boolean. In this case maybe something like activityType: { Commercial, Nonprofit, PartiallyCommercial, Unknown, . . . }

Reply:
OK. There is a spectrum between bringing a "passthru" API into Azure on the one hand, and adding a new Azure API that provides functionality by using someone else's API on the other. You are firmly in the former camp, which basically means feedback on the API isn't usable since you aren't the ones designing it.
It is generally recommended to move toward the second approach: providing functionality in Azure (following Azure standards and patterns), and making use of external APIs as needed/useful. This is not something to enforce in a swagger PR review, but something you should consider at the design stage going forward. We are trying to nudge Azure toward a comprehensive and consistent API surface, and away from a collection of differently-designed APIs and we need long-term help from all of the RPs in order to make progress.